src/ol_infrastructure/applications/airbyte/__main__.py

```diff
@@ -536,6 +536,8 @@
                                 "content": textwrap.dedent(
                                     f"""\
                             ENVIRONMENT={consul_dc}
+                            APPLICATION=data-platform
+                            SERVICE=airbyte
```

Comment:
I would flip these. The application is airbyte and it is a component of the data-platform service.